controllers/admin/AdminLoginController.php

```diff
@@ -201,6 +205,8 @@ public function processLogin()
 					$cookie->last_activity = time();
 
 				$cookie->write();
+				
+				Hook::exec('actionAdminAuthentication');
```

Comment:
The existing hook for auth on front-end is "actionAuthentication". That's why I kept the same pattern with "actionAdminAuthentication".

Do you think we should improve the naming conventions of these hooks?

- actionBeforeAuthentication
- actionAfterAuthenticationFailed (actually actionAuthenticationFailed)
- actionAfterAuthenticationSucceded (actually actionAuthentication)

We'd have to take care of backward compatibilty.

then for Admin:

- actionBeforeAdminAuthentication
- actionAfterAdminAuthenticationFailed
- actionAfterAdminAuthenticationSucceded

What's the best way of naming things, you think?